packages/klevu-core/src/queries/alsoViewed/alsoViewed.ts

```diff
@@ -8,10 +8,12 @@ import { KlevuLastClickedProducts } from "../../store/lastClickedProducts.js"
 
 type Options = {
   limit: number
+  lastClickedProductIds?: string[]
```

Comment:
We need to provide documentation (comment above this) that explains what this feature is. We should also document limit too.